src/error.cs

```diff
@@ -57,6 +57,8 @@
 //		BI1040 The selector {0} on type {1} is found multiple times with different argument out states on argument {2}.
 //		BI1041 The selector {0} on type {1} is found multiple times with different argument types on argument {2} - {3} : {4}.
 //		BI1042 Missing '[Field (LibraryName=value)]' for {field_pi.Name} (e.g."__Internal")
+//		BI1043 Repeated overload {mi.Name} and no [DelegateApiNameAttribute] provided to generate property name on host class.
+//		BI1044 Repeated name '{apiName.Name}' provided in [DelegateApiNameAttribute].
```

Comment:
we don't yet have a a doc file for `BIxxxx` (so it's fine to have the errors just here) but...